lib/redux/actions/pro_mode_wallet_actions.dart

```diff
@@ -92,6 +96,31 @@ ThunkAction startListenToTransferEvents() {
   };
 }
 
+ThunkAction fetchTokensBalances() {
+  return (Store store) async {
+    final logger = await AppFactory().getLogger('action');
+    bool isFetchTokensBalances = store.state.proWalletState?.isFetchTokensBalances ?? false;
+    if (!isFetchTokensBalances) {
+      UserState userState = store.state.userState;
+      new Timer.periodic(Duration(seconds: 3), (Timer timer) async {
```

Comment:
maybe the interval should be in some config?

Reply:
Done!